### ISSUE TITLE
pixelated screen at wake up

### ISSUE THREAD

**Issue by Happy420Palko** [blend-os/blendOS#79]
i have FINALLY been able to get this installed on my old-ish desktop!!  the problem I'm having is when my system wakes from sleep, my screen is so pixelated that i have to reboot by pressing the case button. I am really new at Linux so i will prolly have to get walked thru the gathering of info to make this report of any use to you. so i apologize in advance for that. and thanks you for all your hard work.  ilike this Distro best out of all the ones i have tried so far.

**Comment by highghlow** [comment]
I have the same issue with blendOS. I discovered that after a window gets updated, it mostly works. The only advice I could give you is to disable automatic suspension.

**Comment by Happy420Palko** [comment]
Yes. Thanks. I figured that out as well. Now I'm trying to figure out root permissions. I am unable to view log files. 

Sent from Yahoo Mail on Android 
 
  On Sat, Aug 5, 2023 at 6:23 AM, ***@***.***> wrote:   


I have the same issue with blendOS. I discovered that after a window gets updated, it mostly works. The only advice I could give you is to disable automatic suspension.
